services/scientist/maana-physical-quantity.md

```diff
@@ -502,56 +786,55 @@ schema { query: Query }
 
 | Example | You have a list of strings, each which may contain zero or more entity mentions. How do extract all the physical quantities from the source text only if they have dimensions of length? |
 | :--- | :--- |
-| Info | Use the optional dimensionRef parameter to specify either the id, name or formula of the required physical dimension. If no dimensionRef parameter is provided, the extractor admits all physical dimensions. |
-| Query | { extract\(sources: \[ "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "The minimum runway length is 7500 feet."\], dimensionRef:{name:"LENGTH"}\) {  entity {  name}}} |
-| Response | { "data": { "extract": \[ { "entity": { "name": "3000.0 m"}}, { "entity": { "name": "7500.0 ft"}}\]}} |
+| Info | Use the optional dimensionInputs parameter to specify either the name or formula of the required physical dimension. If no dimensionInputs parameter is provided, the extractor admits all physical dimensions. |
+| Query | { extractBatch\(sources: \[ "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "The minimum runway length is 7500 feet."\], dimensionInputs:\[{name:"LENGTH",formula:"L¹"}\]\) {  entity {  name}}} |
+| Response | { "data": { "extractBatch": \[ { "entity": { "name": "3000.0 m"}}, { "entity": { "name": "7500.0 ft"}}\]}} |
 
 | Example | You have a list of strings, each which may contain zero or more entity mentions. After extracting the entity mentions, how do you know where they occur in the source text? |
 | :--- | :--- |
 | Info | The fromKind, fromField, fromInstance, and fromString fields of the EntityMention type provide locators to the source text. The fromOffset field of the entity EntityMention type provides the distance from the start of the string \(in characters\) to the start of the entity mention. The fromSpan field of the EntityMention type provides the length of the entity mention \(in characters\). |
-| Query | { extract\(sources: \[ "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "The minimum runway length is 7500 feet."\], dimensionRef:{name:"LENGTH"}\) {  entity {  name} fromText, fromSpan, fromOffset}} |
-| Response | { "data": { "extract": \[ { "entity": { "name": "3000.0 m"}, "fromText": "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "fromSpan": "12", "fromOffset": "59"}, { "entity": { "name": "7500.0 ft"}, "fromText": "The minimum runway length is 7500 feet.", "fromSpan": "10", "fromOffset": "29"}\]}} |
+| Query | query LENGTH{physicalDimension\(dimensionRef:{name:"PRESSURE"}\){ ... PhysicalDimensionFragment }}<br>query EXAMPLE($PRESSURE:PhysicalDimensionInput){ extractBatch\(sources: \[ "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "The minimum runway length is 7500 feet."\], dimensionInputs:\[$LENGTH\] \) {  entity {  name} fromText, fromSpan, fromOffset}} |
```

Comment:
I STILL CAN'T GET THIS TO WORK EVEN AFTER READING YOUR DUMB INSTRUCTIONS!!!

Reply:
Looks like there is a typo in the query.   The variable named $PRESSURE is not defined.